db/migrate/20201011201537_add_title_to_movies.rb

```diff
@@ -0,0 +1,5 @@
+class AddTitleToMovies < ActiveRecord::Migration[5.2]
+  def change
+    add_column :movies, :title, :string
```

Comment:
Since title is presumed static for an ID, decided to add title to movies table to avoid extra api calls